pkgs/by-name/mf/mfaktc/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  lib,
+  stdenv,
+  cudatoolkit,
+  autoAddDriverRunpath,
+  cudaPackages,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "mfaktc";
+  version = "0.23.5";
+
+  src = fetchFromGitHub {
+    owner = "primesearch";
+    repo = "mfaktc";
+    tag = "${finalAttrs.version}";
+    fetchSubmodules = true;
+    hash = "sha256-NUcRd+WvmRjXC7rfOKFw4mue7V9oobsy/OTHHEoaiHo=";
+  };
+
+  enableParallelBuilding = true;
+  nativeBuildInputs = [
+    cudatoolkit
```

Comment:
All new projects should use the CUDA redistributables available in [`cudaPackages`](https://search.nixos.org/packages?channel=unstable&type=packages&query=cudaPackages) in place of [`cudaPackages.cudatoolkit`](https://search.nixos.org/packages?channel=unstable&type=packages&query=cudaPackages.cudatoolkit), as they are much easier to maintain and update. (as mentioned in `doc/languages-frameworks/cuda.section.md`), also these should be in buildInputs?